qrisk/stats.py

```diff
@@ -41,17 +40,23 @@
     MONTHLY: MONTHS_PER_YEAR
 }
 
-def cum_returns(returns, starting_value=None):
+
+def cum_returns(returns, starting_value=0):
     """
     Compute cumulative returns from simple returns.
 
     Parameters
     ----------
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+         - Time series with decimal returns.
```

Comment:
With the `returns` metric, do you think it's worth clarifying whether they are % returns or absolute dollar returns? (From the examples, I feel like it's reasonably clear that they are percentages, but wondering if it's worth clarifying anyways)